File: c/leetcode/389. Find the Difference.c


char findTheDifference(char * s, char * t){
    int sum_s = 0;
    int sum_t = 0;
    
    for(int i = 0; i < strlen(s); i++) {
        sum_s += s[i];
    }
    
    for(int i = 0; i < strlen(t); i++) {
        sum_t += t[i];
    }
    
    return sum_t - sum_s;
}

 

File: python/leetcode/389. Find the Difference.py
class Solution:
    def findTheDifference(self, s: str, t: str) -> str:
        sum_s = 0
        sum_t = 0

        for ch in s:
            sum_s += ord(ch)

        for ch in t:
            sum_t += ord(ch)

        return chr(sum_t - sum_s)